### DISCUSSION TITLE
(Device Agent) Start/Restart/Suspend Device from FF editor

### DISCUSSION THREAD

**Discussion by GogoVega** [FlowFuse/flowfuse#3279] [Ideas]
Hi,
Is it planned to be able to control the NR instance on a device from the FF editor?

**Comment by hardillb** [comment]
Hi,

It's something we can have a look at, I can definitely see a use for a restart option, but will need to check how start/stop would work in the current model.

**Reply by GogoVega** [reply]
Great, thanks
